### PR TITLE
chore: Attempt to fix update-aws-cdk script

### PR DESCRIPTION
**Pull request by jacobwinch**
## What does this change?

This PR attempts to fix some of the problems that we noticed in https://github.com/guardian/cdk/pull/641.

### CODE REVIEW

File: script/update-aws-cdk [hunk 1]
```diff
@@ -51,7 +55,7 @@ raisePR() {
 
   git checkout -b $BRANCH_NAME
 
-  for file in package.json package-lock.json integration-test/package.json; do
+  for file in package.json package-lock.json integration-test/package.json integration-test/package-lock.json; do
```
Comment (jacobwinch) [inline script/update-aws-cdk:58]: https://github.com/guardian/cdk/pull/641#pullrequestreview-694701813

>  I think integration-test/package-lock.json should have changed too

Yep, this should fix it!

File: script/update-aws-cdk [hunk 2]
```diff
@@ -34,6 +34,10 @@ updateAwsCdk() {
   npm run update-aws-cdk
 }
 
+installAwsCdkUpdates() {
+  npm install && cd integration-test && npm install
```
Comment (jacobwinch) [inline script/update-aws-cdk:38]: > It also looks like this is a v2 lockfile, which comes from npmv7. Is this desired? 

When this part was being run from within `package.json`, npmv7 was being used. However:
```
npm --version
6.14.8
```
😕

Comment (akash1810) [reply script/update-aws-cdk:38]: 🤔  that _is_ odd!

Comment (akash1810) [reply script/update-aws-cdk:38]: Do we need to `cd ..` after this too to stay at the root? Alternatives:

```sh
(
  cd integration-test
  npm install
)
```

Or `pushd`/`popd`.

**Review by akash1810 — APPROVED** [review]

**Comment by jacobwinch** [comment]
This seems to have worked successfully: https://github.com/guardian/cdk/pull/644.

**Pull request merged**